src/libraries/Microsoft.PowerFx.Core/Syntax/Visitors/ListFunctionVisitor.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+
+namespace Microsoft.PowerFx.Syntax
+{
+    public class ListFunctionVisitor : IdentityTexlVisitor
```

Comment:
>public [](http://example.com/codeflow?start=4&length=6)

internal

Reply:
Sure. I kept it public so we do not need to change the code in other projects.